src/gtclang/Unittest/ParsedString.cpp

```diff
@@ -0,0 +1,47 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                         _       _
+//                        | |     | |
+//                    __ _| |_ ___| | __ _ _ __   __ _
+//                   / _` | __/ __| |/ _` | '_ \ / _` |
+//                  | (_| | || (__| | (_| | | | | (_| |
+//                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+//                    __/ |                       __/ |
+//                   |___/                       |___/
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#include "gtclang/Unittest/ParsedString.h"
```

Comment:
I don't see an `<iostream>` include here.. This usually means you leaked an `<iostream>` from a header file which is not good (https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden obviously it's not forbidden for us but we should still be careful not including it in headers)